app/templates/settings.hbs

```diff
@@ -152,6 +152,38 @@
       {{/unless}}
     </section>
   {{/if}}
+  {{#if this.showBetaFeatures}}
+    <section class="settings-section beta_features">
+      <h2 class="small-title display-inline">
+        Beta Features
+      </h2>
+      <p>
+        These features are in beta. If you find things to be not working as expected
+        or lacking, please let us know in the <a href="https://travis-ci.community">Travis CI community forum</a>
```

Comment:
We have a bunch of URLs in `config/environment.js`, so that we can just have them in one place and easily change if we need to. Looks like one already exists for the community site, so with the`config-get` helper, this link could be `<a href={{config-get 'urls.community}}>`

Reply:
done 🎉